component-server-parent/component-server/src/main/java/org/talend/sdk/component/server/service/jcache/FrontCacheKeyGenerator.java

```diff
@@ -66,7 +65,7 @@ private Stream<Object> getContextualKeys() {
                             headers.getHeaderString(HttpHeaders.ACCEPT_ENCODING));
         } catch (Exception e) {
             log.debug("[getContextualKeys] context not applicable: {}", e.getMessage());
-            return Stream.of(UUID.randomUUID());
+            return Stream.empty();
```

Comment:
Hm, we changed the behavior. 
Not sure how it will influence the flow =)

Reply:
In fact, this fixes a buggy idea for studio side... making never hitting the cache key!